.github/workflows/dependabot-changesets.yml

```diff
@@ -0,0 +1,15 @@
+name: "Dependabot Changesets"
+
+on: pull_request
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  changesets:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository_owner == 'sigstore' &&  github.actor == 'dependabot[bot]' }}
+    steps:
+      - name: Dependabot Changesets
+        uses: feelepxyz/dependabot-changesets@b91eb2125cabe58b66aa7f18d3345855108d6e21 # v1
```

Comment:
Here's an example: https://github.com/feelepxyz/test-dependabot/pull/3/files 

I opted to always create `patch` updates instead of inferring from the semver update of the depenedency or something else clever. Thinking it should be easy enough to just edit the created changelog in the pr files tab if necessary.

Reply:
@bdehamer updated it to create empty changesets for indirect/dev dependencies: https://github.com/feelepxyz/dependabot-changesets/blob/main/src/main.ts

Does this make sense to you? 

I also updated it to now create a separate changelog entry for each changed package if there are multiple and create a summary for the packages updated dependencies: https://github.com/feelepxyz/dependabot-changesets/blob/main/src/main.ts#LL146C7-L146C22 (I suspect there might be cases where this does the wrong thing as I haven't tested this at all on monorepos)